.golangci.yml

```diff
@@ -10,6 +10,7 @@ linters-settings:
   dupl:
     threshold: 100
   exhaustive:
+    check-generated: false
```

Comment:
This is the actual file used for this project so there's no need to add defaults for visibility here.

Reply:
Thanks, updated.